grails-app/assets/javascripts/streama/controllers/dash-ctrl.js

```diff
@@ -151,6 +153,29 @@ angular.module('streama').controller('dashCtrl',
       }
     }
 
+    function getByDashType(dashType) {
+      console.log(dashType);
+      switch (dashType) {
+        case 'home':
+          _.find($scope.settings, {name: 'hidden_dash_sections'}).value = "";
+          $state.go('dash');
+          break;
+        case 'tv_shows':
+          _.find($scope.settings, {name: 'hidden_dash_sections'}).value = "new-releases,continue-watching,recommends,watchlist,discover-movies,discover-generic";
+          break;
+        case 'movies':
+          _.find($scope.settings, {name: 'hidden_dash_sections'}).value = "new-releases,continue-watching,recommends,watchlist,discover-shows,discover-generic";
+          break;
+        case 'watchlist':
+          _.find($scope.settings, {name: 'hidden_dash_sections'}).value = "new-releases,continue-watching,recommends,discover-shows,discover-movies,discover-generic";
+          break;
+        default:
+          _.find($scope.settings, {name: 'hidden_dash_sections'}).value = "";
+          break;
+      }
```

Comment:
This doesnt work yet, and there are no actual links in the header? Please double-check...

Reply:
Fixed and checked - everything works fine.